server_admin/topics/sso-protocols/oidc.adoc

```diff
@@ -241,6 +238,8 @@ This field is optional and was defined by CIBA standard document.
 
 |===
 
+Besides the mentioned parameters, any custom parameter, which is sent to Backchannel Authentication Endpoint, will be forwarded to Channel Provider as well.
```

Comment:
Change it to "parameter that is sent to the Backchannel"

Use "that" because the rest of the sentence further defines the "custom parameter."  Also, omit the comma when using "that" to futher define the item mentioned. Use "which" when the rest of the sentence does not further define the item mentioned and include the comma before "which."

Reply:
Thanks for explaining!